actionmailbox/lib/action_mailbox/errors.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "active_support/actionable_error"
+require "rails/command"
+
+module ActionMailbox
+  # Generic base class for all Action Mailbox exceptions.
+  class Error < StandardError; end
+
+  # Raised when we detect that Action Mailbox has not been initialized.
+  class SetupError < Error
+    include ActiveSupport::ActionableError
+
+    def initialize(message = nil)
+      return super if message
+
+      super("Action Mailbox uses a table in your application’s database named " \
+            "two and active_storage_attachments. To generate it run the following" \
+            "command:\n\n        rails action_mailbox:install")
+    end
+
+    action "Run action_mailbox:install" do
```

Comment:
For the migration task we didn't say "Run db:migrate" but "Run migrations". What format should we follow for these action names?

Reply:
Hm, maybe a phrase will be better here. For example, for the setup of Action Mailbox to be actually complete, we may need to run Active Storage install command as well. The button name won't reflect the real action in that case, so no need to imply it. I'll change this. :+1: